### PR TITLE
e2e (diferent react versions sample): data-e2e locators for diferent react versions sample

### PR DESCRIPTION
**Pull request by PeaceDate**
Screenshot 2022-12-13 at 11 37 03

### CODE REVIEW

File: cypress/fixtures/constants.ts [hunk 1]
```diff
@@ -462,4 +473,8 @@ export class Constants {
         nameField: 'NAME',
         emailField: 'EMAIL'
     }
+
+    public static readonly cssVar = {
+        borderColorRed1px: '1px solid rgb(255, 0, 0)'
```
Comment (edchai) [inline cypress/fixtures/constants.ts]: I suppose could be added to color const

File: different-react-versions/e2e/checkDiferentReactVersionsApp.cy.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            value: Constants.cssVar.borderColorRed1px
+        })
+        basePage.checkElementHaveProperty({
+            selector: block.reactButtonBlock,
+            prop: Constants.commonText.border,
+            value: Constants.cssVar.borderColorRed1px
+        })
+    })
+
+    it('Check App functionality', () => {
```
Comment (edchai) [inline different-react-versions/e2e/checkDiferentReactVersionsApp.cy.ts]: too vague. Please provide more concrete description

**Review by edchai — CHANGES_REQUESTED** [review]

File: cypress/fixtures/constants.ts (hunk 1 above)
Comment (PeaceDate) [reply cypress/fixtures/constants.ts]: Fixed

File: different-react-versions/e2e/checkDiferentReactVersionsApp.cy.ts (hunk 2 above)
Comment (PeaceDate) [reply different-react-versions/e2e/checkDiferentReactVersionsApp.cy.ts]: Fixed

File: cypress/common/selectors.ts [hunk 3]
```diff
@@ -101,6 +101,7 @@ export const alertMessages = {
 }
 
 export const block = {
-    differentReactVersionsIsolatedSharedBlock: '[data-e2e="SHARED__REACT_BLOCK"]'
+    differentReactVersionsIsolatedSharedBlock: '[data-e2e="SHARED__REACT_BLOCK"]',
+    reactDivBlock: '[data-e2e="REACT__DIV_BLOCK"]',
```
Comment (edchai) [inline cypress/common/selectors.ts]: can be merged with button block and get div/button as replace param

**Review by edchai — CHANGES_REQUESTED** [review]

File: cypress/common/selectors.ts (hunk 3 above)
Comment (PeaceDate) [reply cypress/common/selectors.ts]: Fixed

**Review by edchai — APPROVED** [review]

**Pull request merged**